src/lib/coda_base/secure_random.ml

```diff
@@ -1,5 +1,3 @@
 open Core
 
-let () = assert Insecure.randomness
```

Comment:
This was a kind of formalized TODO to make this use dev/urandom or some such. So we should make sure to do that or have a blocking issue or something before removing this

Reply:
We want to use `getrandom` on Linux to wait for the system entropy pool to come up.

Reply:
@enolan I kinda want to just use https://github.com/rust-random/getrandom. Objections?